src/vs/workbench/contrib/notebook/test/browser/notebookEditorModel.test.ts

```diff
@@ -19,6 +19,7 @@ import { CellKind, NotebookData, TransientOptions } from 'vs/workbench/contrib/n
 import { NotebookFileWorkingCopyModel } from 'vs/workbench/contrib/notebook/common/notebookEditorModel';
 import { INotebookSerializer, INotebookService, SimpleNotebookProviderInfo } from 'vs/workbench/contrib/notebook/common/notebookService';
 import { setupInstantiationService } from 'vs/workbench/contrib/notebook/test/browser/testNotebookEditor';
+import { SnapshotContext } from 'vs/workbench/services/workingCopy/common/fileWorkingCopy';
 
 suite('NotebookFileWorkingCopyModel', function () {
```

Comment:
Is it possible to have a test here for the throw on large backup? Maybe by configuring the threshold for the test to a low length for triggering it?